tests/baselines/reference/convertToAsyncFunction/convertToAsyncFunction_NoRes.ts

```diff
@@ -8,7 +8,7 @@ function /*[#|*/f/*|]*/():Promise<void | Response> {
 
 async function f():Promise<void | Response> {
     try {
-        await fetch('https://typescriptlang.org');
+        return await fetch('https://typescriptlang.org');
```

Comment:
Not returning was just a bug?

Reply:
Correct. Passing `null` or `undefined` to the `onfulfilled` handler is essentially the same as passing `v => { return v; }`, the value still flows through.  Similarly, passing `null` or `undefined` to an `onrejected` handler is essentially the same as passing `e => { throw e; }`.